README.rst

```diff
@@ -41,14 +41,21 @@ Optional dependencies
 Installation
 ============
 
-We recommend the `Anaconda Python distribution <https://www.continuum.io/downloads>`_. To install ``hnn-core``, you first need to install its dependencies::
+We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`_.
```

Comment:
Should we mention a recommended Python minor version? I guess that largely depends on what NEURON is built against.

Reply:
I think Neuron doesn't work with 3.9 as far as I know ... I hope Anaconda doesn't distribute bleeding edge Python

Reply:
It might... We could recommend creating an env with python<=3.8? Gets a bit messy though.

Reply:
yeah let's leave it like this for now and see how many users stumble upon that issue